CMakeLists.txt

```diff
@@ -100,7 +100,7 @@ if( NOT HAVE_NO_INSTALL )
     # Setup Python virtual environment
     include( python_venv )
     set( loki_VENV_PATH ${CMAKE_CURRENT_BINARY_DIR}/loki_env )
-    set( PYTHON_VERSION 3.8...<3.12 )
+    set( PYTHON_VERSION 3.8 )
```

Comment:
Shouldn't this be 3.9 now? No need to fix here, just looking for confirmation to fix in a follow-up.

Reply:
I left it intentionally at 3.8 because Loki technically still works with that - so no need to break the integration there. It's just that we can't test it anymore (or at least not with JIT). But I realize I've actually increased the version requirement in pyproject.toml, so it wouldn't work as is. My proposal would be to lower pyproject.toml to 3.8 again until we hit some blocking issue that renders Loki itself (not just the JIT tests) incompatible